src/components/fields/image-upload/image-upload-helper.ts

```diff
@@ -18,4 +18,9 @@ export namespace ImageUploadHelper {
 		const openSlots = possibleSlots.filter((possibleSlot) => !takenSlots.includes(possibleSlot));
 		return openSlots[0] ?? maxSlots;
 	};
+
+	export const assignImageId = (name: string, slot?: number, images?: IImage[]) => {
+		const _slot = slot ?? findAvailableSlot(images);
+		return `${_slot}-${name}`;
```

Comment:
may I suggest to use the random id generator from: https://github.com/LifeSG/web-frontend-engine/blob/main/src/components/fields/file-upload/file-upload-manager.ts#L98

because the image-upload field will be based on the file-upload field eventually and since that is generating id based on that, we want to standardise the random id format.

Reply:
moved the generator to common util file and used it for image upload